vm/src/utils.rs

```diff
@@ -87,15 +87,15 @@ pub mod test_utils {
             num_bigint::BigInt::parse_bytes($val.as_bytes(), $opt).expect("Couldn't parse bytes")
         };
     }
-    pub(crate) use bigint_str;
+    pub use bigint_str;
 
     #[macro_export]
     macro_rules! biguint {
         ($val : expr) => {
             Into::<num_bigint::BigUint>::into($val as u128)
         };
     }
-    pub(crate) use biguint;
+    pub use biguint;
```

Comment:
This is not really vm-related, so I don't think it should be included in the vm's public api. Moreover, we may choose to remove the usage of BigUint from the VM altogether and remove this macro too.